test/specs/files.spec.js

```diff
@@ -203,4 +203,30 @@ describe("bump [files...]", () => {
     expect(files.json("package.json")).to.deep.equal({ version: "1.2.3" });
   });
 
+  it("should skip specified version scripts if present", () => {
```

Comment:
This file is for tests related to the `[files...]` arg.  A better place to put this test would be in the `npm.spec.js` file, which is specifically for tests related to the npm version hooks